weaviate/data/crud_data.py

```diff
@@ -564,6 +569,9 @@ def get(
             _check_positive_num(limit, "limit", int, include_zero=False)
             params["limit"] = limit
 
+        if after is not None:
+            params["after"] = after
```

Comment:
maybe we can check the UUID as well with `weaviate.util.get_valid_uuid()` which extract the UUID from URLs and beacons as well.